composer.json

```diff
@@ -46,10 +46,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require-dev": {
-        "behat/behat": "^3.6",
+        "php": "^8.0",
         "phpstan/extension-installer": "^1.1",
         "phpstan/phpstan": "^0.12.82",
         "phpstan/phpstan-deprecation-rules": "^0.12.6",
+        "rector/rector-src":  "~0.11.0",
```

Comment:
We'll need to revert this now.

Reply:
No, we need this in the PR, it's OK for -dev

Reply:
Ah I see, we could revert it to `rector/rector` for 0.10 vs src

Reply:
But, if I understand correctly, we'd like to add 0.11.0 here in order to work our the integration with phpstan and phpUnit. Is that right?

Since the stable uses 0.10.0 now, we should be fine.